equinix/resource_network_device.go

```diff
@@ -1100,6 +1107,9 @@ func createNetworkDevices(d *schema.ResourceData) (*ne.Device, *ne.Device) {
 	if v, ok := d.GetOk(neDeviceSchemaNames["ClusterDetails"]); ok {
 		primary.ClusterDetails = expandNetworkDeviceClusterDetails(v.([]interface{}))
 	}
+	if v, ok := d.GetOk(neDeviceSchemaNames["Connectivity"]); ok {
+		primary.Connectivity = ne.String(v.(string))
```

Comment:
HA(secondary device) case needs to be updated and tested along with this one.

Reply:
Connectivity field will be applied to both primary and secondary device.

Reply:
@rling-equinix just to confirm, you're saying that we only have to set `primary.Connectivity` because the API will automatically apply that value to both primary and secondary device, and there is no way via the API to set a different value for primary and secondary?

Reply:
Yes, correct.